stable/nginx-ingress/templates/default-deployment.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    component: default-backend
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+  name: {{ template "fullname" . }}-default-backend
+spec:
+  replicas: {{ .Values.default.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+        component: default-backend
+        release: "{{ .Release.Name }}"
+    spec:
```

Comment:
Needs an optional nodeSelector to restrict where it lands in non uniform clusters.